service/tutorial/order/OrderServices.xml

```
@@ -0,0 +1,360 @@
+            <parameter name="orderId" required="true"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.party.Party" list="orderCreateList">
+                <econdition field-name="partyId" value="CustJqp"/>
+            </entity-find>
+            <if condition="orderCreateList">
+                <then>
+                    <service-call name="create#mantle.order.OrderHeader" in-map="context" out-map="context"/>
+                </then>
+            </if>
+        </actions>
+    </service>
+
+
+    <service verb="create" noun="CreateOrderPart" displayName="Create the order Part">
+        <in-parameters>
+            <parameter name="orderId" required="true"/>
+            <parameter name="partName" required="true"/>
+            <parameter name="facilityId" required="true"/>
+            <parameter name="shipmentMethodEnumId" required="true"/>
+            <parameter name="customerPartyId" required="true"/>
+            <parameter name="item_details" type="List">
+                <parameter name="orderEntry" type="Map">
+                    <parameter name="productId"/>
+                    <parameter name="itemDescription"/>
+                    <parameter name="quantity"/>
+                    <parameter name="unitAmount"/>
+                </parameter>
+            </parameter>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orderId" required="true"/>
+            <parameter name="orderPartSeqId" required="true"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderHeader" list="orderCreateList">
+                <econdition field-name="orderId"/>
+            </entity-find>
+            <if condition="orderCreateList">
+                <service-call name="create#mantle.order.OrderPart" in-map="context" out-map="context"/>
+            </if>
+            <entity-find entity-name="mantle.order.OrderPart" list="orderList">
```

Can we return an error if the orderHeader does not exist for the input orderId?

---

Updated recommended changes.
Thank you.